lib/ansible/modules/cloud/azure/azure_rm_aks_info.py

```diff
@@ -14,9 +14,9 @@
 
 DOCUMENTATION = '''
 ---
-module: azure_rm_aks_facts
+module: azure_rm_aks_info
 
-version_added: "2.6"
```

Comment:
just one note here, version_added should stay the same as in old facts module

Reply:
if i don't change the version_added, it cannot pass the check

Reply:
ok

Reply:
I don't think bumping the version is a good idea. In all cases I renamed a module, I never had to do this.